### ISSUE TITLE
Google play link is invalid

### ISSUE THREAD

**Issue by MattixNow** [JakeWharton/Telecine#182]

**Comment by JakeWharton** [comment]
The app was pulled from the store last weekend. Haven't had time to look
into it.

**Comment by andyli** [comment]
Would you publish it on F-droid instead? https://f-droid.org/en/contribute/

**Comment by JakeWharton** [comment]
The app uses closed source software for analytics and as I understand that
is not allowed. Also I don't want to maintain two stores. If the app was
fixed, I would just restore the Play version.

**Comment by MattixNow** [comment]
Thanks

**Comment by IzzySoft** [comment]
@JakeWharton You could provide a build flavor coming without Google Analytics, Firebase and GMS. F-Droid could then build that flavor (that's what several apps use). If you don't want "no analytics", you also could replace GA by e.g. ACRA or Matomo. Until you finished that build variant, find your app [in my repo](https://apt.izzysoft.de/fdroid/index/apk/com.jakewharton.telecine) (which in a way is to F-Droid what "nonfree" is to Debian).

**Comment by JakeWharton** [comment]
I'm not interested in maintaining two versions. I'm not even maintaining
one right now.

**Comment by IzzySoft** [comment]
In that case, if it's an option for you, you could simply remove the 3 "offending libraries", @JakeWharton. I'd be surprised if you'd agree, yes. But I like good surprises :rofl:

**Comment by IzzySoft** [comment]
@JakeWharton soon's one year gone, and the app still hasn't returned even on Play. Will it come back at all?

**Comment by JakeWharton** [comment]
Probably not.

**Comment by IzzySoft** [comment]
Sad… Not even a final release with all the changes made since the last one, added here at Github (in `releases/`)? And if you won't develop it further, maybe even with Analytics dropped?

Whatever your decision will be: All the best for whatever you're on now!

**Comment by JakeWharton** [comment]
With the new audio recording in Q maybe I'll find it in me to do one more...

**Comment by IzzySoft** [comment]
Great! Looking forward to that!

**Comment by Kisty** [comment]
Was it pulled due to the GDPR data privacy EU law? If so, you could add a toggle for in-app tracking in the settings or something.

**Comment by JakeWharton** [comment]
No. It was something else. I don't even remember at this point.

**Comment by IzzySoft** [comment]
Just wondering, @JakeWharton – are you still planning to revive it? Last release was 3 years ago. The app is still in my repo; but should I rather mark it "unmaintained"?

**Comment by JakeWharton** [comment]
The version is your repo was never maintained by me, so you're free to mark it however you like.

**Comment by IzzySoft** [comment]
I don't want to be a liar, saying "no longer maintained" if in fact you were maintaining it still and there just wasn't a release, @JakeWharton :wink: So could you please give a yes or no on the question whether you still do maintain the app? Maybe the correct term would be "currently not **actively** maintained" – meaning you didn't abandon it but currently cannot spend much time on it either?

**Comment by JakeWharton** [comment]
That seems accurate, yes. I aspire to maintain it, but alas, other things
often prevail when prioritizing my time.

**Comment by IzzySoft** [comment]
Thanks, Jake – added a corresponding comment (takes effect with tomorrow's sync). All the best – and we look forward to whenever you find some time again for the app!
